### PR TITLE
[4.10] Fix each invalid config.xml

### PR DESCRIPTION
**Pull request by gitseti**
https://hivemq.kanbanize.com/ctrl_board/4/cards/10370/details/

### CODE REVIEW

File: hivemq4/dns-image/config-dns.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@
         <enabled>true</enabled>
         <discovery>
             <extension>
-                <reload-interval>${HIVEMQ_DNS_DISCOVERY_INTERVAL}</reload-interval>
```
Comment (hurtadosanti) [inline hivemq4/dns-image/config-dns.xml:34]: Why did you remove this?

File: hivemq4/k8s-image/config.xml [hunk 2]
```diff
@@ -26,13 +26,10 @@
             <tcp>
                 <bind-address>${HIVEMQ_BIND_ADDRESS}</bind-address>
                 <bind-port>${HIVEMQ_CLUSTER_PORT}</bind-port>
-                <enabled>true</enabled>
             </tcp>
         </transport>
         <discovery>
-            <extension>
```
Comment (hurtadosanti) [inline hivemq4/k8s-image/config.xml:33]: Why did you remove this?

File: hivemq4/dns-image/config-dns.xml (hunk 1 above)
Comment (gitseti) [reply hivemq4/dns-image/config-dns.xml:34]: This tag does actually not exist in the hivemq config.

File: hivemq4/k8s-image/config.xml (hunk 2 above)
Comment (gitseti) [reply hivemq4/k8s-image/config.xml:33]: The tag is still present. I just removed the reload interval as it prevents the hivemq startup on 4.10.

**Review by schaebo — DISMISSED** [review]

File: hivemq4/dns-image/config-dns.xml [hunk 3]
```diff
@@ -31,7 +31,7 @@
         <enabled>true</enabled>
         <discovery>
             <extension>
```
Comment (schaebo) [inline hivemq4/dns-image/config-dns.xml:33]: could be <extension/> instead

File: hivemq4/k8s-image/config.xml (hunk 2 above)
Comment (SgtSilvio) [reply hivemq4/k8s-image/config.xml:33]: `reload-interval` never did anything, it is an invalid tag, so can be removed safely

File: hivemq4/dns-image/config-dns.xml (hunk 3 above)
Comment (SgtSilvio) [reply hivemq4/dns-image/config-dns.xml:33]: please replace 
```
<extension>
</extension>
```
with 
```
<extension/>
```

**Review by schaebo — APPROVED** [review]

File: hivemq4/dns-image/config-dns.xml (hunk 1 above)
Comment (hurtadosanti) [reply hivemq4/dns-image/config-dns.xml:34]: I think the HIVEMQ_DNS_DISCOVERY_INTERVAL should be removed from the documentation as well then, the information is on the extension documentation.

Comment (gitseti) [reply hivemq4/dns-image/config-dns.xml:34]: I don't think this should go into this PR, as we also have other env variables for the dns-discovery-extension only in the README, which then should be removed as well and replaced by a link to the dns-cluster-discovery-extension.
Also, the Dockerfile currently overrides the default of 30 for the interval with 31, so I would not want to introduce a behavior change with this.

**Review by hurtadosanti — APPROVED** [review]

**Review by SgtSilvio — APPROVED** [review]
